docs/usage/about-us.md

```diff
@@ -37,6 +37,7 @@ Some features made a lot of people happy, and efficient!
 - [@astellingwerf](https://github.com/astellingwerf) reviews PRs
```

Comment:
Question for the maintainers: do we redirect links to GitHub profiles in our release notes code? I want to prevent `@setchy` from getting link-back spammed by GitHub when show users the Renovate release notes with their username in it. 😄

Reply:
Yes, we try to